### PR TITLE
fix yarn start

### PR DESCRIPTION
**Pull request by ahmadzakiy**
fix the issue after cloning the repo and run command `yarn start` cause error when starting docs site. because of radix still not build.

### CODE REVIEW

**Comment by ahmadzakiy** [comment]
> Thanks for this. Out of curiosity, what does the `init` flag do differently?

It will build `@modulz/radix-system` and `@modulz/radix` first, before running storybook and docs site in parallel.

in my case, after fresh clone and run `yarn start` , I got this error.
Screen Shot 2020-02-23 at 04 37 04

**Comment by peduarte** [comment]
Oh I see, to confirm, did you: 
1. Clone 
2. Run `yarn`
3. Run `yarn start`

Cheers

**Comment by ahmadzakiy** [comment]
yeah, I'd followed that step

**Comment by peduarte** [comment]
I see, I think I prefer to update the docs, to explain that you need to run `npm init` when you first clone it. 

It's something that only needs to be done once, after a flesh clone.

This is because when gatsby tries to build, there's no build artefacts of Radix. The same for when Radix tries to built, there'd no build artefacts of Radix System.

When dev'ing locally, running `npm start` puts everthing in watch mode.

But I dont think we should run a build as part of `start`, it's going to slow local development by a lot.

Thanks for the suggestion though 🙏 

If you can to update the Read me, I'd accept the PR. 

---

**NOTE**: very important note by the way, Radix is still in Alpha and it's not recommended to be used. There will be many more breaking changes coming, under the same alpha flag. Use at your own risk ⚠️

**Comment by ahmadzakiy** [comment]
make sense, I'm trying to update readme, Thank you 🙏

File: README.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ This repository is a [monorepo](https://github.com/babel/babel/blob/master/doc/d
 yarn
 ```
 
+#### Build Radix library
```
Comment (peduarte) [inline README.md]: I'd rename it to this, since it also builds radix-system

```suggestion
#### Initialise after a first install
```

**Comment by peduarte** [comment]
Awesome, thanks for the contribution 😉

**Pull request merged**